src/FINDOLOGIC/Api/Responses/Xml21/Properties/LandingPage.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use SimpleXMLElement;
+
+class LandingPage
+{
+    /** @var string $link */
+    private $link;
+
+    /**
+     * LandingPage constructor.
+     * @param SimpleXMLElement $response
+     */
+    public function __construct($response)
```

Comment:
Please typehint this with `SimpleXMLElement` and drop the PHPDoc as it is not really needed anyway.